src/libs/mergeRequestDownloader.ts

```diff
@@ -0,0 +1,39 @@
+import { browser } from 'webextension-polyfill-ts';
+import { MergeRequestsDetails } from '../background/types';
+
+const REQUEST_TYPE_GET_MERGE_REQUESTS = 'getMRs';
+
+export interface MergeRequestSendMessageReply {
+    mrAssigned: MergeRequestsDetails[];
+    mrToReview: number;
+    mrGiven: MergeRequestsDetails[];
+    mrReviewed: number;
+    lastUpdateDateUnix: number;
+    error?: string;
+}
+
+export const getMergeRequestList = async (): Promise<MergeRequestSendMessageReply> => {
```

Comment:
Please consider writing functions with Callback as in the other files to be a bit uniform.

Reply:
or have everything aync/await :)

Reply:
Well I love callbacks :joy: 

And if you didn't see, the async library (in the background script) is awesome :+1: